xcode/Subconscious/Shared/Components/Common/Transclude/TranscludeView.swift

```diff
@@ -2,41 +2,95 @@
 //  TranscludeView.swift
 //  Subconscious
 //
-//  Created by Gordon Brander on 8/23/22.
+//  Created by Gordon Brander on 2/22/23.
 //
 
 import SwiftUI
 
+struct ExcerptView: View {
+    var excerpt: String
+    var lineLimit: Int = 5
+    var spacing: CGFloat = AppTheme.unit
+    var excerptLines: [EnumeratedSequence<[String.SubSequence]>.Element] {
+        Array(excerpt.split(separator: "\n").enumerated())
+    }
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: spacing) {
+            ForEach(excerptLines, id: \.offset) { idx, line in
+                Text("\(String(line))")
+                    .fontWeight(idx == 0 && excerptLines.count > 1 ? .medium : .regular)
+                    .opacity(idx > 0 ? 0.8 : 1) // Helps massively in dark mode
+                    .lineLimit(lineLimit)
```

Comment:
I think we don't want to set a line limit on the transclude. Otherwise we get a strange effect where each line gets truncated.

Seems ok to let these get a bit long.

Edge case: we could set some high upper limit or truncation, as a sanity check against blocks that are > 50 lines long.